src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/TableParsers/IlrTableParser.cs

```diff
@@ -96,8 +96,8 @@ private static IlrTableStructure ParseTableStructure(Table ilrDetails)
                     case "residual training price 1":
```

Comment:
ditto the other table parsers - can these for loops with switch statements in be simplified

Reply:
In ParseCommitmentsTableRow method:

Can include RestartIndicator, LearningAdjustmentForPriorLearning and OtherFundingAdjustments in intialiser
lines 185, 186, 188, 193, 210 - generic argumnet not required.